python/plugins/db_manager/db_plugins/vlayers/connector.py

```diff
@@ -342,14 +342,22 @@ def runVacuum(self):
         print("**unimplemented** runVacuum")
         return False
 
+    def commentTable(self, schema, tablename, comment=None):
+        """Comment the table"""
+        return ''
+
+    def getComment(self, tablename, field):
```

Comment:
Move this method `getComment` into the base class `DBConnector` as a default implementation and override in PG connector only, instead of providing the same default implementation in all connectors but the PG one.